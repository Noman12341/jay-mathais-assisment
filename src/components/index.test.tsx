import React from 'react';
import { mount, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Component from './index';

describe('Root component', () => {
  it('should match snapshot', () => {
    const el =   shallow(<Component />);
    expect(el.exists()).toBe(true);
  });
});