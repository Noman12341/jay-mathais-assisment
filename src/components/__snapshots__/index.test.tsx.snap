// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root component should match snapshot 1`] = `
    <Paper className={classes.root}>
      <Box className={classes.searchBtnBox}>
        <FormControl className={classes.formControl}>
          <InputLabel id="CountryInput">Select City</InputLabel>
          <Select labelId="cityLable" id="city" value={city} onChange={(e: any) => setCity(e.target.value)}>
            <MenuItem value="Tokyo">Tokyo</MenuItem>
            <MenuItem value="Delhi">Delhi</MenuItem>
            <MenuItem value="Shanghai">Shanghai</MenuItem>
            <MenuItem value="Sao Paulo">Sao Paulo</MenuItem>
            <MenuItem value="Mexico City">Mexico City</MenuItem>
          </Select>
        </FormControl>
      </Box>
    </Paper>
`;
